.gitignore

```diff
@@ -17,4 +17,9 @@ __pycache__
 /tests/bdd/autogen/
 /rust-toolchain.toml
 /.rust-toolchain
-/kubectl-plugin
\ No newline at end of file
+/kubectl-plugin
```

Comment:
Is this where you download the kubectl-plugin binary?
I'd suggest keeping a directory for all downloaded binaries to keep it clean

Reply:
I didn't put it there. It was there when I found it, the new-line at the end is making it seem like a new line. Not a binary download for minikube.

Reply:
hmm oh it's for the nix-build, if you don't specify arguments it puts them on current directory :D